src/update-repo.ts

```diff
@@ -176,7 +176,7 @@ async function createAndMergePullRequest({
   const res = await octokit.pulls.create({
     ...repo,
     head: branch,
-    base: "master",
+    base: "main",
```

Comment:
**suggestion**: As mentioned in [this PR](https://github.com/artsy/update-repo/pull/52), I think we should change these hard-coded default values in an isolated PR that triggers a major version bump, since it would be a breaking change.